### PR TITLE
Bugfix trilinos application values

### PR DESCRIPTION
**Pull request by sunethwarna**
Fixing small bugs in trilinos and metis application.

### CODE REVIEW

File: applications/metis_application/custom_processes/metis_divide_heterogeneous_input_in_memory_process.h [hunk 1]
```diff
@@ -239,11 +239,11 @@ class MetisDivideHeterogeneousInputInMemoryProcess : public MetisDivideHeterogen
             GraphColoringProcess(mNumberOfPartitions,DomainGraph,ColoredDomainGraph,NumColors).Execute();
 
             if (mVerbosity > 0) {
-                KRATOS_INFO(NumColors);
+                KRATOS_WATCH(NumColors);
```
Comment (philbucher) [inline applications/metis_application/custom_processes/metis_divide_heterogeneous_input_in_memory_process.h:242]: @sunethwarna I'd say the fix in #2617 is better here

File: kratos/solving_strategies/convergencecriterias/residual_criteria.h [hunk 2]
```diff
@@ -142,7 +142,7 @@ class ResidualCriteria
         const TSystemVectorType& b
         ) override
     {
-        const SizeType size_b = b.size();
+        const SizeType size_b = TSparseSpace::Size(b);
```
Comment (philbucher) [inline kratos/solving_strategies/convergencecriterias/residual_criteria.h:145]: I think the same problem exists in the displacement_criteria

File: applications/metis_application/custom_processes/metis_divide_heterogeneous_input_in_memory_process.h (hunk 1 above)
Comment (sunethwarna) [reply applications/metis_application/custom_processes/metis_divide_heterogeneous_input_in_memory_process.h:242]: I agree :) I can revoke this pull request. You have everything handled in #2617. Only issue will be the changes I made in residual criteria.

File: kratos/solving_strategies/convergencecriterias/residual_criteria.h (hunk 2 above)
Comment (sunethwarna) [reply kratos/solving_strategies/convergencecriterias/residual_criteria.h:145]: ping @jcotela

**Comment by sunethwarna** [comment]
Leaving changes to #2617

**Comment by philbucher** [comment]
I just wanted to approve it...

**Comment by sunethwarna** [comment]
I made changes to #2617 :)

**Pull request closed without merging**